project/users/management/commands/create_groups.py

```diff
@@ -0,0 +1,36 @@
+from django.core.management.base import BaseCommand
+from django.contrib.auth.models import Group, Permission
+from django.contrib.contenttypes.models import ContentType
+from project.partners.models import Partner
+from project.program.models import Activity, Presenter
+
+PERMISSIONS = ['view', 'edit', 'add', 'delete']
+permissions = {'media': ['partner'], 'fr': ['partner'],
```

Comment:
`permissions_per_group` maybe? to avoid confusion with `PERMISSIONS`

Reply:
Yes, you are right, too many permissions there.